docs/internals/contributing/bugs-and-features.txt

```diff
@@ -10,8 +10,10 @@ Reporting bugs and requesting features
     not public. For further details, please see :doc:`our security
     policies </internals/security>`.
 
-Otherwise, before reporting a bug or requesting a new feature on the
-`ticket tracker <https://code.djangoproject.com/>`_, consider these points:
+Otherwise, before reporting a bug on the
+`ticket tracker <https://code.djangoproject.com/>`_, or a new feature on the
+`new feature ideas tracker <https://github.com/orgs/django/projects/24/>`_
+consider these points:
```

Comment:
I think the points that come after are very Trac specific, you might want to update them for the new feature ideas tracker to make sense